packages/zosjobs/src/GetJobs.ts

```diff
@@ -153,6 +153,7 @@ export class GetJobs {
         Logger.getAppLogger().trace("GetJobs.getJobsCommon()");
         ImperativeExpect.toNotBeNullOrUndefined(session, "Required session must be defined");
         let query = JobsConstants.QUERY_ID;
+        params.execData = true; // always returning start and end time data
```

Comment:
If a user explicitly turns off the execData flag ( for example --no-exec-data), I think that this statement will overwrite the user's choice. Maybe we need to check whether execData does not have a value of false, before attempting to return the new exec properties. Maybe we should set some local variable to reflect that determination and use that variable in later decisions in this function?

Admittedly, this seems like an unlikely user action, but it appears to be currently possible for a user to ask for no execData.

Reply:
I resolved this. Let me know if it's good.